opennms-container/minion/CONFD_README.md

```diff
@@ -161,3 +161,33 @@ system:
 ```
 Config specified will be written to `etc/confd.system.properties` which gets automatically appended to `etc/system.properties`. Additionally, provided the
 `jaeger-agent-host` key is specified, `etc/featuresBoot.d/jaeger.boot` will also be updated.
+
+### Karaf Properties
+```
+---
+karaf:
+    shell:
+        ssh:
+            host: "0.0.0.0"
+            port: 8201
+    management:
+        rmi:
+            registry:
+                host: "127.0.0.1"
+                port: 1299
+            server:
+                host: "127.0.0.1"
+                port: 45444
+```
+Config specified will be written to:
+- `etc/org.apache.karaf.shell.cfg` for content under `shell`.
+- `etc/org.apache.karaf.management.cfg` for content under `management`.
+
+### Jolokia Properties
+```
+---
+jolokia:
```

Comment:
Should rename this to `jetty` - it's also used by Hawtio and other webapps.

Reply:
Sounds good; re-named.